### PR TITLE
Update NeuRAD page with slider functionality

### PR DESCRIPTION
**Pull request by atonderski**
This pull request includes several changes to the NeuRAD page:

- Initial edits to NeuRAD page

- Avoid need to specify correct aspect ratio in 'two_image_slider'

- Add more content to NeuRAD

- More text on NeuRAD

- Use arxiv icon

- link sliders

The changes include adding slider functionality to the page, updating the aspect ratio of the slider images, and adding more content to the page.

### CODE REVIEW

**Comment by atonderski** [comment]
maybe merge with squash since we added and deleted som images?

File: _publications/neurad/neurad.md [hunk 1]
```diff
@@ -14,8 +14,79 @@ code: https://github.com/georghess/neurad
...
+  width="35"
+  id="2"
+  linkid="1"
+  caption="Effect of rolling shutter on depth"
+%}
+</div>
+
+## Ray divergence
+
+Recent work has shown that NeRFs suffer from aliasing artifacts when objects in the scene are viewed at different distances. This is especially common in automotive scenes, where the sensors (ego vehicle) moves through the scene, and the distance to the objects in the scene changes. Inspired by recent work (ZipNeRF), we model the beam divergence of the lidar sensor, and the size of individual pixels, by reweighting the iNGP hashgrid features. Specifically, for a given ray sample, we compare the volume of the frustum it represents, with the volume of a voxel at each resolution of the hashgrid. We then downweight features for scales that are much smaller than the considered frustum. This is illustrated below:
```
Comment (carlinds) [inline _publications/neurad/neurad.md:74]: Add references

Comment (carlinds) [inline _publications/neurad/neurad.md:74]: Add references

File: _publications/neurad/neurad.md [hunk 2]
```diff
@@ -14,8 +14,79 @@ code: https://github.com/georghess/neurad
...
+{% include two_image_slider.html
+  left_image="without_appemb.jpg"
+  right_image="with_appemb.jpg"
+  width="45"
+  id="3"
+  caption="Rendering without and with sensor embedding"
+%}
+</div>
+
+Note that this is inspired by recent work on NeRFs in the wild, with two key differences. First, we only learn a single embedding per sensor, not per image, improving generalizaiton to novel views. Second, we apply the sensor embedding after the volume rendering, reducing the computational overhead. This is not possible in the original work, as they directly render RGB, whereas we render high-level features.
```
Comment (carlinds) [inline _publications/neurad/neurad.md:92]: Add references

**Review by georghess — CHANGES_REQUESTED** [review]

File: _publications/neurad/neurad.md [hunk 3]
```diff
@@ -14,8 +14,79 @@ code: https://github.com/georghess/neurad
...
+
+---
+
+# Method
+
+<figure>
+  <img style="width: 100%; margin: 0 auto;" src="MethodOverview.png"/>
+</figure>
+
+The core of NeuRAD is a Neural Feature Field (NFF), a generalization of NeRFs and similar approaches. The NFF is a function \\(F(\mathbf{x}, t, \mathbf{d})\\) that maps a 4D point \\(\mathbf{x_i}, t_i\\), and a direction \\(\mathbf{d_i}\\) to a geometry descriptor \\(s_i\\) and a feature vector \\(\mathbf{f_i}\\). We then use volume rendering to generate per-ray features \\(\mathbf{f}\\) and expected depth. For lidar rays we use a small MLP to predict the probability that the ray did not return (ray drop), as well as the intensity of the return. For camera rays, we arrange features into a patch, and use a small ConvNet to predict the RGB color of the patch.
```
[the remark this answers is not in the document]
Comment (atonderski) [reply _publications/neurad/neurad.md]: you meant 3D point at a current time right?

[the remark this answers is not in the document]
Comment (georghess) [reply _publications/neurad/neurad.md]: Ah, yeah

[the remark this answers is not in the document]
Comment (atonderski) [reply _publications/neurad/neurad.md]: great, that's how I did it

**Review by georghess — APPROVED** [review]

**Comment by georghess** [comment]
LGTM

**Pull request merged**